src/handlers/invoke-private-api.ts

```diff
@@ -0,0 +1,52 @@
+import { AppConfigService } from '../services/app-config-service';
+import logger from '../commons/logger';
+
+interface CustomEvent {
+  userId?: string;
+  queryParameters?: string;
+  headers?: {
+    [key: string]: string;
+  };
+}
+
+const appConfig = AppConfigService.getInstance();
+
+export const handle = async (event: CustomEvent) => {
+  const userId = event.userId ?? appConfig.userId;
```

Comment:
is there a reason why the userId can be set as a environment variable? The default value through the env var is an empty string, which wouldn't really work as a default value. Would it not be safer to enforce that a userid is passed in the event?

Reply:
Yeah, we wanted it to be possible to use this lambda using either the lambda event or the environment variables.

Reply:
Okay.. I think the env var make sense for things like the endpoint, which are not going to change often. But I really don't understand including the userId and queryParameters in the env vars, sorry - is this how this lambda is going to be used in the test framework?